### PR TITLE
Profile Pics Are A-Rockin'

### PR DESCRIPTION
**Pull request by seanboom**
Can't you hear them knockin'?
Better not be blockin'!
Or you go belly of croc in!

### CODE REVIEW

File: src/Components/ProfilePic.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  },
+
+  getInitialState() {
+    return {
+      error: false
+    };
+  },
+
+  componentWillReceiveProps(nextProps) {
+    console.log('yada');
```
Comment (mfunkie) [inline src/Components/ProfilePic.jsx]: Remove this.

**Comment by mfunkie** [comment]
I think it's good otherwise.

**Comment by mfunkie** [comment]
Looks good, wait for the build to complete and merge it, then publish

**Comment by seanboom** [comment]
Agh. I merged before the build completed. Your comment didn't appear -- I was just going off what you told me in person.

**Pull request merged**